lib/http/2/compressor.rb

```diff
@@ -359,15 +349,15 @@ def set_table_size(size)
       # @return [String] binary string
       def integer(i, n)
         limit = 2**n - 1
-        return [i].pack('C') if (i < limit)
+        return [i].pack('C') if i < limit
 
         bytes = []
-        bytes.push limit if !n.zero?
+        bytes.push limit unless n.zero?
```

Comment:
Maybe it's just me, but my brain has a _really_ hard time grokking `unless`.. I can read the `if !expr` without any issues, but I always stumble across `unless` and have to reparse it a few times before it makes sense. 

What do the various style guides recommend for unless? Personally, I'd prefer to avoid it, but maybe that's just me. /cc @mad-p